packages/apps/kadena-docs/src/pages/index.tsx

```diff
@@ -1,13 +1,37 @@
 import { Stack } from '@kadena/react-components';
 
+import { IDocsPageFC } from '@/types/Layout';
 import React from 'react';
 
-export default function Home(): JSX.Element {
+const Home: IDocsPageFC = () => {
   return (
-    <Stack>
-      <div>1</div>
-      <div>2</div>
-      <div>3</div>
-    </Stack>
+    <>
```

Comment:
Is this Fragment a requirement of MarcDoc or just a “left-over”?

Reply:
will remove